lib/net_suite/configuration.rb

```diff
@@ -8,11 +8,13 @@ class Configuration
       :logger!,
       {
         log_requests: false,
+        datadog_request_tracing: false,
```

Comment:
How about this simpler, but still accurate, name?
```suggestion
        datadog_tracing: false,
```

Reply:
That's fine by me. Any objections @sbader?

Reply:
@ericgross I went with `datadog_request_tracing` since the PR just got approved with that spelling. We can follow up if you feel strongly